src/main/resources/db/migration/common/V1_280__add_regions.sql

```diff
@@ -0,0 +1,9 @@
+INSERT INTO region (reg_id, region_name) VALUES(1, 'London');
```

Comment:
this file's version needs to be bumped up now, we have files on master with versions past this one

Reply:
I've bumped them up again.